src/wwwroot/app/services/NoteService.js

```diff
@@ -12,12 +16,17 @@ export default class NoteService {
    }
 
    save(entity) {
+      this.convertDateToServerFormat(entity);
       if (entity.id) {
-         return _HttpService.put(`${NOTE_URL}/${entity.id}`, entity);
+         return _HttpService.put(`${NOTE_URL}/${entity.id}`, entity).then((comment) => {
```

Comment:
you can call method in 'then' instead of creating a new clause. like 
**_HttpService.put(`...).then(this.convertDateFromServerFormat)**